controllers/kluctl_project.go

```diff
@@ -748,29 +752,38 @@ func (pt *preparedTarget) kluctlDeploy(ctx context.Context, targetContext *kluct
 
 	cmdResult, err := cmd.Run(ctx, targetContext.SharedContext.K, nil)
 	err = pt.handleCommandResult(ctx, err, cmdResult, "deploy")
+	internal_metrics.NewKluctlNumberOfImages(pt.pp.obj.Namespace, pt.pp.obj.Name).Set(float64(len(cmdResult.SeenImages)))
```

Comment:
This could be moved into the function I mentioned above

Reply:
See above